doc/content/examples/pore_scale_transport/index.md

```diff
@@ -0,0 +1,308 @@
+
+!media examples/figures/2D_pore_structure_closed.png
+  id=fig:initial_image_closed
+  caption= Initial image used to generate the microstructure with closed pores. This corresponds to the original image to generate Fig. 5.a in [!cite](Simon2022).
+  style=display:block;margin-left:auto;margin-right:auto;width:40%;border-style:solid
+
+!media examples/figures/2D_pore_structure_open.png
+  id=fig:initial_image_open
+  caption= Initial image used to generate the microstructure with open pores. This corresponds to the original image to generate Fig. 5.b in [!cite](Simon2022).
+  style=display:block;margin-left:auto;margin-right:auto;width:40%
+
+TMAP8 can import these images using [ImageFunction](ImageFunction.md) and, using a threshold, attribute different contrast values to given phases.
+Here, the bright areas are identified as ceramic material, while dark areas are identified as pores.
+
+The size of the microstructure is not directly provided by the images in [fig:initial_image_closed] and [fig:initial_image_open].
+Users can scale, crop, and adjust the image to the desired size.
+In this example, the sample being 4.5 mm in radius, the domain size is set to 5.425 mm $\times$ 5.425 mm.
+
+!listing test/tests/pore_scale_transport/2D_microstructure_reader_smoothing_base.i link=false block=Mesh
+
+!listing test/tests/pore_scale_transport/2D_microstructure_reader_smoothing_base.i link=false block=Functions
+
+#### Perform a phase field simulation to smooth the interfaces
+
+!style halign=left
+The rest of the input file is set up as a [phase field](phase_field/index.md) simulation to smoothen the interface between the pore and the ceramic material.
+As discussed in [!cite](Simon2022), using a phase field approach with a continuous interface enables TMAP8 to model arbitrarily complex geometries that cannot be easily meshed.
+Since TMAP8 inherits the [phase field module](phase_field/index.md) from [MOOSE], it can perform phase field simulations.
+
+In this case, we use a two-phase model with arbitrary parameter values to quickly obtain, from the pore structures illustrated in [fig:initial_image_closed] and [fig:initial_image_open], corresponding microstructures with the desired interface width, i.e., $l$=50 $\mu$m in this case.
+The simulation time is selected to (1) be large enough to obtain a smooth interface, while (2) staying small enough to prevent significant changes in the microstructure.
+
+#### Run simulations
+
+To run the input file to reproduce the microstructures shown in Fig. 5 in [!cite](Simon2022), users need to first [install TMAP8](installation.md). Then, to generate the microstructure with closed pores, run:
+
+```
+cd ~/projects/TMAP8/test/tests/pore_scale_transport/
+mpirun -np 4 ~/projects/TMAP8/tmap8-opt -i 2D_microstructure_reader_smoothing_base.i pore_structure_closed.params
```

Comment:
Give a brief mention about running this on 4 CPU cores using `mpirun`. Those familiar with running TMAP8 serially might not be as familiar with running it in parallel.